ui/app/src/components/UppyDashboard/UppyDashboard.tsx

```diff
@@ -442,7 +443,7 @@ export function UppyDashboard(props: UppyDashboardProps) {
       title,
       id: 'craftercms:Dashboard',
       site,
-      path,
+      path: ensureSingleSlash(`${path}/`),
```

Comment:
What's the need for this?

Reply:
`path` is the base path that's used in the dashboard. It may come without a slash at the end, so when appending the rest of the path (in the case of folder structures) it was causing the full path to be incorrect.
For example
- path = `/static-assets/images`
- upload folder structure like `folder1/folder2/image.jpg`
was ending up in: `/static-assets/imagesfolder1/folder2/image.png`